aten/src/ATen/templates/TypeDefault.cpp

```diff
@@ -31,6 +31,36 @@ namespace TypeDefault {
 
 TORCH_LIBRARY(aten, m) {
   ${function_registrations};
+  m.def("splitlines(str self, bool keepends=False) -> str[]");
```

Comment:
Probably would be good to add a little comment here saying what these registrations are (and where they are registered)